frontend/packages/ceph-storage-plugin/src/utils/create-storage-system.ts

```diff
@@ -0,0 +1,11 @@
+import { ExtensionK8sModel } from '@console/dynamic-plugin-sdk/src';
+import { SUPPORTED_EXTERNAL_STORAGE } from '../components/create-storage-system/external-storage';
+import { WizardState } from '../components/create-storage-system/reducer';
+
+export const getStorageSystemKind = ({ kind, version, group }: ExtensionK8sModel) =>
+  `${kind.toLowerCase()}.${group}/${version}`;
```

Comment:
Why not keep this similar to what we have with the rest of the console `group~version~kind` and instead convert the value that we receive from the CR? That would help up interact with the k8s helpers better.

Reply:
@bipuladh that is the spec for StoargeSystemKind, I do not control that.

Reply:
I understand what you're doing. You are converting this to the format used by ODF. But rest of the console uses `group~version~kind` format. Instead of us converting to them. We can convert values passed by ODF when using in console. That would allow us to use utilities used to work with models efficiently.

Reply:
The `kind` property of storage system CR accepts the kind to be of this format, hence its required.